src/components/TrainingComponents/TrainingVideo.tsx

```diff
@@ -1,46 +1,88 @@
-import * as React from "react";
-import { DefaultPlayer as Video } from "react-html5video/dist";
-import "react-html5video/dist/styles.css";
-import * as PropTypes from "prop-types";
+import * as React from 'react'
+import { DefaultPlayer as Video } from 'react-html5video/dist'
+import 'react-html5video/dist/styles.css'
+import * as PropTypes from 'prop-types'
+import { addWatchedVideos } from '../../state/actions/watched-videos'
+import { connect } from 'react-redux'
 
 interface Props {
-  videoUrl: string
+    videoUrl: string
+    watchedVideos: Array<string>
+}
+
+interface State {
+    watchedVideos: Array
```

Comment:
`State` interface seems to have invalid keys in it, let's fix them to match the proper state object keys
```suggestion
    autoPlay: boolean
```